index.html

```diff
@@ -6558,13 +6558,13 @@ <h2 id="defining-a-valid-audioworkletprocessor">
               <ul>
                 <li>Nodes that transform their inputs, and are active only
                 while connected inputs and/or script references exist. Such
-                nodes should return <code>false</code> from
+                nodes SHOULD return <code>false</code> from
```

Comment:
The other bullets seem to say SHOULD, at least in the current PR I'm looking at (lines 6567 and 6576).